static/sass/styles.scss

```diff
@@ -1383,3 +1383,15 @@ hr.p-separator.is-shallow {
 
   background-color: #fff;
 }
+
+.u-aspect-ratio--3-2 {
```

Comment:
this class already exists in line 1330 https://github.com/canonical/ubuntu.com/blob/df5ac10e2bc2436412efe73c64842d47e3fa4667/static/sass/styles.scss